src/components/helpers.js

```diff
@@ -43,6 +43,17 @@ import {
   OTHER_PURPOSE
 } from 'src/constants/const'
 
+export const modes = [
+  AIR_MODE,
```

Comment:
What is "air" mode? Does this include all aerial transports like plane, helicopter etc?

Reply:
Aucune idée, pour l'instant on est sur de l'avion uniquement je pense